visualization/app/codeCharta/util/fileNameHelper.ts

```diff
@@ -34,7 +34,7 @@ export class FileNameHelper {
 		return fileName
 	}
 
-	static withoutCCJsonExtension(fileName: string) {
-		return fileName.replace(/(\.cc)?\.json?$/, "")
+	static withoutCCExtension(fileName: string) {
+		return fileName.replace(/(\.cc)?(\.json)?(\.gz)?$/, "")
```

Comment:
Hadn't we agreed to only replace `(\.json)?(\.gz)?$`?

Reply:
For now we will remove all extensions, like it is mentioned in the original issue #2568 . So old files are sill displayed correctly